packages/core/src/internal/validation/futures/validateNamedContractDeployment.ts

```diff
@@ -65,7 +65,8 @@ export async function validateNamedContractDeployment(
 
   const missingParams = moduleParams.filter(
     (param) =>
-      deploymentParameters[param.moduleId]?.[param.name] === undefined &&
+      (deploymentParameters[param.moduleId]?.[param.name] === undefined &&
+        deploymentParameters.$global[param.name]) === undefined &&
```

Comment:
I think this is a bug, the additional brackets are not going to give the value expected.